pset-functions.js

```diff
@@ -135,11 +159,61 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+function scrabbleScore(word) {
+  // creates an array with each letter as a string
+  const letter = word.split("")
+  let letterOccurence = {}
+
+  //NEED to transfrom letters into lowercase
+  //goes through each element in the letter array
+  //assigns a value for the letter to a key in the letterValue object
+  letter.forEach((element) => {
```

Comment:
@kyleligman same issue re: structuring your conditionals from https://github.com/jsd20190730/hw-02-functions-and-iterators/pull/4/files#r320092922